### DISCUSSION TITLE
Can we pin older versions of LSPs?

### DISCUSSION THREAD

**Discussion by DennisTheMenace780** [mason-org/mason.nvim#894] [Q&A] [answered]
I'm using a version of Ruby that is out of date with the minimum requirements of both `ruby-lsp` and `solargraph`. I am running `ruby 2.6.8p205` but the min requirements for both LSPs seem to be `ruby 2.7.0`. So while unfortunate, I need to get a different commit of the LSPs but i'm not sure if we can specify this in Mason or not.

**Comment by williamboman** [comment, marked as answer]
Yes it's possible, see https://github.com/williamboman/mason.nvim/discussions/880#discussioncomment-4643467
